pwiz_tools/Skyline/TestData/CommandLineTest.cs

```diff
@@ -646,6 +660,137 @@ public void ConsoleOverwriteDocumentTest()
             }
         }
 
+        [TestMethod]
+        public void ConsoleModsTest()
+        {
+
+            TestFilesDir = new TestFilesDir(TestContext, ZIP_FILE);
+            string docPath = TestFilesDir.GetTestPath("BSA_Protea_label_free_20100323_meth3_multi.sky");
+            
+            // test --pep-add-mod, --pep-add-unimod, --pep-add-mod-aa, and --pep-add-mod-term
+            {
+                var mods = new Dictionary<object, StaticMod>
+                {
+                    // either long name or tuple with short/unimod name, AA, and terminus
+                    {"Acetyl (N-term)", null},
+                    {"Phospho (ST)", null},
+                    {"Acetyl:13C(2) (K)", null},
+                    {("Oxi", "M", ""), UniMod.GetModification("Oxidation (M)", out _)},

```

Comment:
If the full name "Oxidation", "M" is supported, it should be tested here, or it should be noted that it is not supported and added to the error cases at the end.

Reply:
Not supported (yet) as discussed above.